src/github.com/stellar/gateway/compliance/handlers/request_handler_tx_status.go

```diff
@@ -0,0 +1,81 @@
+package handlers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/stellar/gateway/protocols"
+	"github.com/stellar/gateway/server"
+)
+
+// TransactionStatusResponse represents a response from the tx_status endpoint
+type TransactionStatusResponse struct {
+	protocols.SuccessResponse
+	Status   string `json:"status"`
+	RecvCode string `json:"recv_code,omitempty"`
+	RefundTx string `json:"refund_tx,omitempty"`
+	Msg      string `json:"msg,omitempty"`
+}
```

Comment:
I think we should move this struct to [github.com/stellar/go/tree/master/protocols/compliance](https://github.com/stellar/go/tree/master/protocols/compliance). We plan to move bridge and compliance to monorepo anyway so for new things we should probably make use of it.

Reply:
Is there an equivalent of `protocols.SuccessResponse` in the monorepo? Moving it as it is will still require an import from `"github.com/stellar/gateway/protocols"` is that optimal?

Reply:
OK, let's update `stellar/go` in glide and use types from there.